src/ripple/ledger/PaymentSandbox.h

```diff
@@ -57,10 +67,12 @@ class DeferredCredits
     // lowAccount, highAccount
     using Key = std::tuple<
         AccountID, AccountID, Currency>;
-
-    // lowAccountCredits, highAccountCredits
-    using Value = std::tuple<
-        STAmount, STAmount>;
+    struct Value
+    {
+        STAmount lowAccCredits;
+        STAmount highAccCredits;
+        STAmount lowAccOrgBalance;
```

Comment:
Tiny quibble: personally, "Acc" and "Org" don't quickly clue me into intent.  I'm guessing that "Acc" relates to "Account"?  If so, perhaps "Acct" would give a better clue than "Acc" with only one additional letter.  Is "Org" intending "Original"?  "Orig" may be a more common abbreviation for "Original" if that's what you're after: https://www.allacronyms.com/original/abbreviated.